nss-day-cohort-46/sprinkles-of-joy-jawa-juice-cakes
# Eb sprinkles reviews

# Description

Added a write a review button, created our review form.

Fixes # (issue)

added feature no issues

Please delete options that are not relevant.

- [x ] New feature (non-breaking change which adds functionality)

# Testing Instructions
Please check in your browser.
```
git fetch all
git checkout EB-Sprinkles-Reviews
```


# Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
